src/components/FlatmapVuer.vue

```diff
@@ -2527,22 +2543,22 @@ export default {
       taxonConnectivity: [],
       pathwaysMaxHeight: 1000,
       hoverVisibilities: [
-        { value: false, ref: 'zoomInPopover' }, // 0
-        { value: false, ref: 'zoomOutPopover' }, // 1
-        { value: false, ref: 'zoomFitPopover' }, // 2
-        { value: false, ref: 'settingsPopover' }, // 3
-        { value: false, ref: 'checkBoxPopover' }, // 4
-        { value: false, ref: 'markerPopover' }, // 5
-        { value: false, ref: 'warningPopover' }, // 6
-        { value: false, ref: 'whatsNewPopover' }, // 7
-        { value: false, ref: 'openMapPopover' }, // 8
+        { value: false, ref: 'markerPopover' }, // 0
+        { value: false, ref: 'zoomInPopover' }, // 1
+        { value: false, ref: 'zoomOutPopover' }, // 2
+        { value: false, ref: 'zoomFitPopover' }, // 3
+        { value: false, ref: 'openMapPopover' }, // 4
+        { value: false, ref: 'settingsPopover' }, // 5
+        { value: false, ref: 'checkBoxPopover' }, // 6
+        { value: false, ref: 'warningPopover' }, // 7
+        { value: false, ref: 'whatsNewPopover' }, // 8
         { value: false, ref: 'featuredMarkerPopover' }, // 9
-        { value: false, refs: 'drawToolPopover', ref: 'connectionPopover' }, // 10
-        { value: false, refs: 'drawToolPopover', ref: 'drawPointPopover' }, // 11
-        { value: false, refs: 'drawToolPopover', ref: 'drawLinePopover' }, // 12
-        { value: false, refs: 'drawToolPopover', ref: 'drawPolygonPopover' }, // 13
-        { value: false, refs: 'drawToolPopover', ref: 'deletePopover' }, // 14
-        { value: false, refs: 'drawToolPopover', ref: 'editPopover' }, // 15
+        { value: false, refs: "toolbarPopover", ref: "editPopover" }, // 10
+        { value: false, refs: "toolbarPopover", ref: "deletePopover" }, // 11
+        { value: false, refs: "toolbarPopover", ref: "pointPopover" }, // 12
+        { value: false, refs: "toolbarPopover", ref: "lineStringPopover" }, // 13
+        { value: false, refs: "toolbarPopover", ref: "polygonPopover" }, // 14
+        { value: false, refs: "toolbarPopover", ref: "connectionPopover" }, // 15
       ],
```

Comment:
Is there any reason to change the order? This has caused some issues with displaying the help tooltip.

Reply:
Hmm, the upstream merge has broken something. Nevermind.